### PR TITLE
CCP endpoint

### PR DESCRIPTION
**Pull request by gregquinn2001**
JIRA 1449. CCP endpoint which shows summary and treatment options sections only and no notes shown

## Submitter:

**Running the Application**

- [X] Manually tested in Chrome
- [ ] Manually tested in IE

**Documentation**

- [X] This pull request describes why these changes were made
- [X] Recognizes any potential shortcomings/bugs in the description 

**Code Quality**

- [X] 4-space indents - convert any tabs to spaces
- [X] Use public class field syntax for binding functions - ex. handleChange = () => { ... };
- [X] Code is commented

**Tests**

- [X] Existing tests passed

## Reviewer 1: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

## Reviewer 2: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by mgramigna** [comment]
[MG1] App breaks when choosing fracture as condition:

image

**Comment by mgramigna** [comment]
[MG2] It might be a little misleading having the cursor be a pointer when hovering over the structured data, suggesting that clicking the data will do something, whereas in this case it will do nothing since we don't have any notes or anything

**Comment by gregquinn2001** [comment]
Thanks @mgramigna  Great feedback!

**Comment by mgramigna** [comment]
Small code thing, but we can probably remove these lines right? Since we don't care about the note viewer

https://github.com/FluxNotes/flux/blob/a2e586b22e27e8d7607fb582269d7de4eeb3226c/src/dashboard/CoreCancerPilotDashboard.jsx#L8-L44

**Comment by gregquinn2001** [comment]
Yes! Will do. Call that MG3.

**Comment by gregquinn2001** [comment]
Addressed MG1 and MG3. Also put demo on name of treatment options section

**Comment by jafeltra** [comment]
This looks good! One really small thing I ran into:

[JA1] Hovering over search results and trying to scroll to them causes an error because there is no minimap. (Uncaught TypeError: Cannot read property 'moveToSubsection' of null at TargetedDataPanel.moveToSubsection (TargetedDataPanel.jsx:20))

File: src/dashboard/CoreCancerPilotDashboard.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                        ref={(tdp) => { this.targetedDataPanel = tdp; }}
+                        searchIndex={this.props.searchIndex}
+                    />
+                </div>
+            </div>
+        );
+    }
+}
+
+CoreCancerPilotDashboard.proptypes = {
```
Comment (jafeltra) [inline src/dashboard/CoreCancerPilotDashboard.jsx]: [JA2] Also pretty small, but propTypes should be camel cased in order to actually do prop type checking. This also happens in CoreCancerPilotApp.jsx.

**Comment by gregquinn2001** [comment]
addressed MG2. will look at JA1 and JA2 now!

**Comment by gregquinn2001** [comment]
Addressed JA1 and JA2

**Comment by jafeltra** [comment]
[JA3] MG2 still happens for the Narrative Visualizer.

**Comment by gregquinn2001** [comment]
JA3 fixed. MG-3 JA-3. all tied up. I guess we'll need a tie breaker between Matt and Julia now too!

**Pull request merged**